src/common.cc

```diff
@@ -81,12 +89,13 @@ stim::DetectorErrorModel common::merge_identical_errors(const stim::DetectorErro
       case stim::DemInstructionType::DEM_ERROR: {
         Error error(instruction);
         assert(error.symptom.detectors.size());
```

Comment:
Really we should handle this case correctly, but in the meantime we should at least ensure we don't crash the python interpreter:

```suggestion
        if (error.symptom.detectors.size() == 0) {
            throw std::invalid_argument("Errors that do not flip any detectors are not supported.");
        }
```